### PR TITLE
Fixes annotations JSON to YAML for service-monitor

### PR DESCRIPTION
**Pull request by KateFiroozi**
#3180

### CODE REVIEW

File: charts/promtail/templates/service-metrics.yaml [hunk 1]
```diff
@@ -7,8 +7,11 @@ metadata:
   labels:
     {{- include "promtail.labels" . | nindent 4 }}
     {{- toYaml .Values.service.labels | nindent 4 }}
+  {{- end }}
```
Comment (ein-stein-chen) [inline charts/promtail/templates/service-metrics.yaml:10]: The additional labels should probably also be inside an `{{- if …}}`, or?

```suggestion
  labels:
    {{- include "promtail.labels" . | nindent 4 }}
    {{- with .Values.service.labels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```

**Review by ein-stein-chen — CHANGES_REQUESTED** [review]

**Comment by Sheikh-Abubaker** [comment]
Closing this on behalf of #3164

**Comment by Sheikh-Abubaker** [comment]
Closing this on behalf of #3164

**Review by Sheikh-Abubaker — COMMENTED** [review]
Closed this on behalf of #3164

**Pull request closed without merging**